docs/tutorial/getting-started.md

```diff
@@ -1,3 +1,215 @@
+```
+
+This creates a simple layer containing only one service (which runs a basic HTTP server using Python's `http` module, listening on port 8080).
+
+## Start the Pebble daemon
+
+Now we are ready to run the Pebble daemon.
+
+Pebble is invoked using `pebble <command>`. (To get more information, run `pebble -h`.)
+
+To start the daemon, run:
+
+```bash
+pebble run
+```
+
+This starts the Pebble daemon itself, as well as all the services that are marked as `startup: enabled` (in our simple layer created above, the `http-server` service is marked as `startup: enabled`). You should get some output similar to the following:
+
+```bash
+2024-06-02T11:30:02.925Z [pebble] Started daemon.
+2024-06-02T11:30:02.936Z [pebble] POST /v1/services 10.751704ms 202
+2024-06-02T11:30:02.936Z [pebble] Started default services with change 77.
+2024-06-02T11:30:02.942Z [pebble] Service "http-server" starting: python3 -m http.server 8080
+...
+```
+
+As you can see from the log, our HTTP server is started too, which can be verified by running `curl localhost:8080` in another terminal tab.
+
+> Note: To exit the Pebble daemon, press Ctrl-C (which sends an "interrupt" signal to the process).
+
+## View, start and stop services
+
+While the Pebble daemon is running, you can view the status of services by running `pebble services`. Open another terminal tab, and run:
+
+```bash
+pebble services
+```
+
+You should see output similar to the following:
+
+```bash
+Service      Startup  Current  Since
+http-server  enabled  active   today at 11:30 UTC
+```
+
+Use `pebble stop <service1> <service2> ...` to stop one or more services. You can stop the running `http-server` service by running:
+
+```bash
+pebble stop http-server
+```
+
+You should get output similar to the following:
+
+```bash
+Service      Startup  Current   Since
+http-server  enabled  inactive  today at 11:33 UTC
+```
+
+Now the service `http-server` is stopped. If we run `curl localhost:8080` again, we get a "connection refused" error, which confirms the service is down.
+
+To start it again, run:
+
+```bash
+pebble start http-server
+```
+
+## Add a new layer
+
+Now let's add another layer containing a different service that is also an HTTP server. To create a new layer, run:
+
+```bash
+echo """\
+summary: Simple layer 2
+
+description: |
+    Yet another simple layer.
+
+services:
+    http-server-2:
+        override: replace
+        summary: demo http server 2
+        command: python3 -m http.server 8081
+        startup: enabled
+""" > $PEBBLE/layers/002-another-http-server.yaml
+```
+
+This creates another layer that also contains a single service (running a basic
+HTTP server using the Python `http.server` module listening on a different port 8081).
+
+Add the new layer to a Pebble plan by running:
+
+```bash
+pebble add layer1 $PEBBLE/layers/002-another-http-server.yaml
+```
+
+If the layer is added successfully, the above command should produce the following output:
+
+```bash
```

Comment:
Same comment about using the terminal output directive.
https://github.com/canonical/pebble/pull/421/files#r1635663342